globals.ts

```diff
@@ -45,6 +45,12 @@ export const FormattingPatterns = {
 	 * The `id` group property is present on the `exec` result of this expression
 	 */
 	Role: /<@&(?<id>\d{17,20})>/,
+	/**
+	 * Regular expression for matching a application command mention
+	 *
+	 * The `name` and `id` group properties is present on the `exec` result of this expression
+	 */
+	ApplicationCommand: /<\/(?<name>\w{1,32}):(?<id>\d{17,20})>/,
```

Comment:
The name part of the regex is still not accurate

https://github.com/discordjs/discord.js/blob/fe34f48efbb0a6fd138826b5a4753f39ac2f1ece/packages/builders/src/interactions/slashCommands/Assertions.ts#L11

Also: 
- discordjs/discord.js/pull/8265

Reply:
That regular expression doesn't seem to satisfy slash command mentions, from testing. It probably only really works with `/w{1,32}`.

Reply:
`\w` is not sufficient as it also supports dashes, and subcommands (and groups)

I would suggest something like this:
```suggestion
	ApplicationCommand: /<\/(?<fullName>(?<name>[\w-]{1,32})(?: (?<subcommandOrGroup>[\w-]{1,32}))?(?: (?<subcommand>[\w-]{1,32}))?):(?<id>\d{17,20})/,
```

Reply:
Shouldn't names use `[\P{Lu}\p{N}\p{sc=Devanagari}\p{sc=Thai}_-]{1,32}`?

Reply:
Shouldn't names use `[\P{Lu}\p{N}\p{sc=Devanagari}\p{sc=Thai}_-]{1,32}`?

Reply:
@almeidx could you add the `>` into your code suggestion? Besides that, looks great 👍

Reply:
Oops. Done